.gitignore

```diff
@@ -38,3 +38,4 @@ node_modules/
 # Generated files when building with make container-build
 .config/
 .npm/
+.hugo_build.lock
```

Comment:
as mentioned in the review comment, please move this file to a separate PR

Reply:
Since the line doesn't do anything anyway, I just removed it. Figured it wasn't worth the extra overhead on either of us.